core/src/main/java/org/frankframework/jms/JmsMessagingSourceFactory.java

```diff
@@ -99,7 +97,7 @@ protected ConnectionFactory createConnectionFactory(Context context, String cfNa
 			if (connectionFactoryInfo==null) {
 				connectionFactoryInfo = connectionFactory.toString();
 			}
-			log.info(jmsFacade.getLogPrefix()+"looked up connection factory ["+cfName+"]: ["+connectionFactoryInfo+"]");
+			log.info("%slooked up connection factory [%s]: [%s]".formatted(jmsFacade.getLogPrefix(), cfName, connectionFactoryInfo));
```

Comment:
Volgens mij is `{}` net iets handiger ivm casting, maar dit zou ook moeten werken.

Reply:
De reden om `{}` te gebruiken is dat het lazy is, dit wordt nu altijd gedaan ook als de regel niet gelogd wordt.

Reply:
Ah ja, helder. Ik zal het meenemen in een volgende PR, zodat deze nu door kan (hoop ik).